tox.ini

```diff
@@ -110,10 +110,13 @@ changedir = {toxinidir}
 deps =
     {env:LSR_ANSIBLE_DEP:ansible}
     docker{env:LSR_MOLECULE_DOCKER_VERSION:}
-    molecule<3
+    molecule
     selinux
     wcwidth==0.1.9;python_version=="3.6"
     -rmolecule_extra_requirements.txt
+deps_extra_lint =
+    ansible-lint
+    flake8
```

Comment:
The molecule test envs use the same `envdir`. AFAIU, this will make them all use the same package set so there seems to be no value to specify the dependencies for linting separately. A commit might be nice to explain why they are needed, though.